apps/books/templates/categoryvisebooks.html

```diff
@@ -0,0 +1,28 @@
+<h1>{{ object.title }}:</h1>
+<ul style="list-style-type: none;">
```

Comment:
Please use external sheet for css everywhere. You ll have better idea to handle static assets after this.. and also will have better structure